int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -411,24 +412,6 @@ function edqSetPhoneValidationConfiguration() {
 * Sets the configuration to use Global Intuitive
 */
 function edqSetGlobalIntuitiveConfiguration() {
-	/**
-	* Autofill not working on billing address at all
-	* By doing this in the checkout stage for SFRA we can remove the global-intuitive-unicorn.js in order to force to reload all the content with 
-	* the new parameters that we're including, since the webpage doesn't do a refresh items we're not changing values 
-	* For more information see Bug #126164 */
-	$("script[src=\"" + edqGlobalIntuitiveUnicornJsPath + "\"]").remove();
-	$("<script>").attr({
-		src:edqGlobalIntuitiveUnicornJsPath,
-		integrity:"sha512-ooVQYWcrVoGAZXC+qPMJaFsEkLB82EsT42J+p0U5INWm+NHrN+XvnWsALGv440Zg3a/QsDJ2L9XR0592fkSBmA==",
-		crossorigin:"anonymous"
-	}).appendTo("footer");
-	if(edqAddressLine1Id) {
```

Comment:
Deleted this "if" because it was reloading the window.EdqConfig, and making the index to be lost when the keys up and down where pressed